src/algebra/ordered_field.lean

```diff
@@ -655,4 +656,17 @@ by rw [abs_div, abs_one]
 lemma abs_inv (a : α) : abs a⁻¹ = (abs a)⁻¹ :=
 (abs_hom : monoid_with_zero_hom α α).map_inv' a
 
+lemma one_div_pow_strict_mono_decr_on : strict_mono_decr_on (λ x : α, 1 / x) (set.Ioi 0) :=
```

Comment:
The name doesn't seem to match the statement here - there's no `pow` in there? Also it might be nice to have a version of this lemma with `⁻¹` rather than `1 /`?

Reply:
Bhavik, you are right: I changed the name, by removing the extra `pow`.  I left a comment with a `todo` for the `inv` part: I am in the middle of the `ordered` refactor and most of the lemmas here will have to be adapted.  Thus, I would rather prove these directly with the newer assumptions, than introduce them now and then see if the work again later...

Reply:
Sounds great!